src/histolab/exceptions.py

```diff
@@ -33,13 +33,17 @@ def __str__(self):
         return ""
 
 
+class FilterCompositionError(HistolabException):
+    """Raised when a filter composition for the class is not available"""
+
+
 class LevelError(HistolabException):
     """Raised when a requested level is not available"""
 
 
-class FilterCompositionError(HistolabException):
-    """Raised when a filter composition for the class is not available"""
+class MayNeedLargeImageError(HistolabException):
```

Comment:
Why not only `LargeImageError`?

Reply:
I prefer to keep it explicit as ``LargeImageError`` gives the impression this is coming from the ``large_image`` package whereas in reality it's due to the potential need for large_image to be used.